apps/condo/domains/common/components/tasks/TasksContextProvider.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useEffect, useState } from 'react'
+import { notification } from 'antd'
+import filter from 'lodash/filter'
+import identity from 'lodash/identity'
+import findIndex from 'lodash/findIndex'
```

Comment:
```
import { filter, identity, findIndex } from 'lodash'
```

Reply:
`lodash` provides module separated imports that would be better for decrease production bundle size

Reply:
`lodash` provides module separated imports that would be better for decrease production bundle size